FluentScheduler/Scheduler/Schedule.cs

```diff
@@ -17,17 +17,68 @@ public class Schedule
         /// </summary>
         /// <param name="job">Job to be scheduled</param>
         /// <param name="cronExpression">The scheduling as a cron expression</param>
-        public Schedule(Action job, string cronExpression) =>
+        public Schedule(Action job, string cronExpression) : this(() => MakeAsync(job), cronExpression)
+        {
+        }
+
+        /// <summary>
+        /// Creates a new schedule for the given job.
+        /// </summary>
+        /// <param name="job">Job to be scheduled</param>
+        /// <param name="specifier">The scheduling as a fluent call</param>
+        public Schedule(Action job, Action<RunSpecifier> specifier) : this(() => MakeAsync(job), specifier)
+        {
+        }
+
+        /// <summary>
+        /// Creates a new schedule for the given job.
+        /// </summary>
+        /// <param name="job">Job to be scheduled</param>
+        /// <param name="cronExpression">The scheduling as a cron expression</param>
+        public Schedule(Func<Task> job, string cronExpression) : this((ct) => job(), cronExpression)

```

Comment:
Minor thing: rest of the project calls it `token`.

Reply:
Fixed.